dashboard/test/models/section_test.rb

```diff
@@ -84,19 +84,15 @@ class SectionTest < ActiveSupport::TestCase
   end
 
   test 'name is required' do
-    assert_does_not_create(Section) do
-      section = Section.new user: @teacher
-      refute section.valid?
-      assert_equal ['Name is required'], section.errors.full_messages
-    end
+    section = Section.new user: @teacher
```

Comment:
OIC. I also didn't initially realize this `Section.new` was from the existing code. I'm fine either way.